rundeckapp/grails-app/controllers/rundeck/controllers/FrameworkController.groovy

```diff
@@ -3500,5 +3510,15 @@ class FrameworkController extends ControllerBase implements ApplicationContextAw
         }
         render(status: HttpServletResponse.SC_NO_CONTENT)
     }
+
+    @CompileStatic
+    private def nodeSourceFileExceededMaxSize(ByteArrayInputStream bais){
+        def nodesMaxSize = 65472
```

Comment:
* This would limit the file size to 64kb. How was this limit decided? 
IMO it should be not equal but a little less than the value at https://github.com/rundeck/rundeck/blob/b9db4cf0ff290a9bc88141412627d89382868db3/rundeckapp/grails-app/domain/rundeck/Storage.groovy#L44

* Also this value cannot be hardcoded here. It should be at least moved to a constant so its easier to find and mantain.

Reply:
Hey Alberto, thanks for helping out to improving this code; Regarding this comment, although we state in the ORM mappings that the max size of that attribute will be '_n_' quantity, when we connect rundeck with an external DB (like mysql) GORM creates for that attribute, the column DATA with BLOB datatype, which can hold up to 65,535 bytes of data. That is because we declared that limit.